### DISCUSSION TITLE
Home Assistant setup doesn’t complete with Republic

### DISCUSSION THREAD

**Discussion by jmp2000** [mampfes/hacs_waste_collection_schedule#3294] [Q&A]
Hello - installed 2.5.0 through HACS the other day.  I select United States, Republic Waste, enter my address.   Waits about 5 minutes and then didn't complete the setup. The address is typed exactly as shown on the Republic website, so i know that part should work... is there some other add-on I need in HA to enable this great integration?  Thanks!

**Comment by 5ila5** [comment]
It does work on my side for our test cases (see below)

There was a fix not yet part of the current release, which fixed an error affecting only a very limited amount of addresses (as far as I can tell, none of our test cases failed before the fix.) #3173

But if your address would be affected by this issue, you should see an error message in the config screen.

Do you see anything in your HA logs (setting -> system -> logs)?

```json
{
    "Scott Country Clerk": {
        "street_address": "101 E Main St, Georgetown, KY 40324",
        "method": 1,
    },
    "Branch County Clerk": {
        "street_address": "31 Division St. Coldwater, MI 49036",
        "method": "1",
    },
    "Contract Collection": {
        "street_address": "8957 Park Meadows Dr, Elk Grove, CA 95624",
        "method": 2,
    },
    "Residential Collection": {
        "street_address": "117 Roxie Ln, Georgetown, KY 40324",
        "method": "2",
    },
    "No Method arg": {
        "street_address": "8957 Park Meadows Dr, Elk Grove, CA 95624",
    },
    "4420 Oasis Hill Ave, North Las Vegas, NV 89085": {
        "street_address": "4420 Oasis Hill Ave, North Las Vegas, NV 89085",
        "method": 2,
    },
}
```

**Comment by jmp2000** [comment]
Thanks for the reply!

Nothing in the Home Assistant logs.

I removed it from HACS, rebooted, and reinstalled the integration.   I get
this error.   Any ideas?   Any prerequisites that i need to install
beforehand?

[image: image.png]

**Reply by 5ila5** [reply]
I cannot see the png. Maybe you answered through email?

**Reply by jmp2000** [reply]
Apologies - sure did.  Here's the error message I got after a fresh removal, reboot, installation.   The address does work on the Republic website so I'm pretty sure it's not something with the commas or state abbreviations, etc.

![Screenshot 2024-12-22 105339](https://github.com/user-attachments/assets/633d4792-2e34-4794-9032-24ae68bac68f)

**Reply by jmp2000** [reply]
Good morning - just getting a few minutes to keep troubleshooting this one.   Got the latest version, tried all various reboots....    Is there any other programs that I need installed on a Home Assistant system to use this integration?
